src/Components/MovieDetails/MovieDetails.module.css

```
@@ -0,0 +1,62 @@
+.container {
+  display: flex;
+  background-color: #232323;
+  color: white;
+  padding: 20px;
+  border-radius: 8px;
+  max-width: 800px;
+  margin: auto;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans',
+    'Droid Sans', 'Helvetica Neue', sans-serif;
+}
+
+img {
+  border-radius: 8px;
+  max-width: 300px;
+  margin-right: 30px;
+}
+
+.movieName {
+  font-size: 36px;
+  font-weight: 300;
+  margin-bottom: 10px;
+}
+
+.releaseYear,
+.duration {
+  color: #f65261;
+  font-size: 18px;
+  margin-bottom: 5px;
+}
+
+.releaseYear {
+  margin-right: 50px;
+}
+
+.detailsContainer {
+  display: inline-flex;
+}
+
+.description {
+  color: #787878;
+  font: 'Montserrat';
```

Did you mean `font-family`?

```suggestion
  font-family: 'Montserrat';
```

And it's good practice to add some backup families like you did above.